src/utils.js

```diff
@@ -0,0 +1,90 @@
+go.utils = function () {
+  var _ = require('lodash');
+  var vumigo = require('vumigo_v02');
+  var HttpApi = vumigo.http.api.HttpApi;
+
+  function format_address(address) {
+    if (isNoU(address) || address.length === 0) {
+      return '';
+    }
+
+    var len = address.length;
+    var last_4_digits = address.slice(-4, len);
+    return (len > 4 ? Array(len - 3).join('*') : '') + last_4_digits;
+  }
+
+  function send_email(email_config, usr_addr, im) {
+    var options = {
+      headers: {
+        'Authorization': 'Bearer ' + email_config.api_key,
+        'Content-Type': 'application/json'
+      }
+    };
+
+    var body = {
+      "personalizations": [{
+        "to": [{
+          "email": email_config.to
+        }],
+        "subject": email_config.subject
+      }],
+      "from": {
+        "email": email_config.from,
+        "name": email_config.from_name
+      },
+      "template_id": email_config.template,
+      "custom_args": {
+        "user_address": usr_addr
+      }
+    };
+
+    var api = new HttpApi(im, options);
+
+    return api.post('https://api.sendgrid.com/v3/mail/send', {
+        body: body
+      })
+      .then(function (response) {
+        console.log(response);
```

Comment:
You can't use `console.log()` in production. You can either just remove these if you no longer need them, or use `im.log(..)` if you do still need them.